strip_binaries.py

```diff
@@ -5,23 +5,50 @@
 import subprocess
 
 basepath = sys.argv[1]
+debugroot = sys.argv[2]
+if (debugroot[-1] == os.path.sep) and (len(debugroot) > 1):
+    debugroot = debugroot[:-1]
 
 files = {}
 
 paths = [basepath]
 
-# def read_retval(command):
-#     pr = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-#     retval = pr.returncode
-#     if retval != 0:
-#         data = None
-#     else:
-#         data = pr.stdout.decode('utf8')
-#     return retval, data
+def read_retval(command):
+    # command must be a list of strings
+    pr = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
if you're find with a `subprocess` call, then it all looks just translation of the nasty bash I made into python, so it should be all fine

Reply:
Yes, that's basically what I did.

Reply:
which is totally fine :)

Reply:
I was considering using an ELF module, but I preferred to be safe. But if you think that it would be better, I can do the change...

Reply:
It might, but honestly is it worth right now ? I'm unsure.

Reply:
Well... depending on the output of FILE can be dangerous (as you already said, it is a must to set "LC_ALL=C" to avoid problems), so maybe it is better...

Reply:
I wont block on it at least. I was wondering if a clean design would not be a snapcraft-level plugin, but I dont know anything about all of that and maybe it is a dumb idea.

Reply:
Ok, I used pyelftool.

Reply:
@lissyx Can you review it and tell me what do you think?

Reply:
@sergio-costas I already r+'d it, and I had another look after seeing your push of `pyelftool` changes and I dont see anything that bothers me

Reply:
Thanks!